CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+
+#### Table of Contents
+
+[Getting Started](#getting-started)
+  * [Working Together](#working-together)
+  * [Project Goals](#project-goals)
+  * [The Gonum Repositories](#the-gonum-repositories)
+
+[Contributing](#Contributing)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Code Contribution](#code-contribution)
+  * [Code Review](#code-review)
+  * [What Can I Do to Help?](#what-can-i-do-to-help)
+  * [Style](#style)
+
+## Getting Started
```

Comment:
I'm unclear on what this is for. With the text as it is, this seems like a heading for a heading's sake. Would "Overview" or "The Project" or something like that be better (I don't like either of these overmuch).